sql/dql/aggregate-queries/sql-practice-aggregate-single-table-2.md

```diff
@@ -15,9 +15,9 @@ type: exercise
 
 linkType: sqlfiddle
 
-link: http://sqlfiddle.com/#!9/b3fa3a
+link: http://sqlfiddle.com/#!9/b3fa3a/58
```

Comment:
ok. I'm having a really hard time with sqlfiddle rn (it takes minutes to load). Can we table these older ones until Monday since we have brand new basic exercises now?